src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1825,4 +1825,18 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
   <data name="FancyZones_Zone_Appearance.Header" xml:space="preserve">
     <value>Zone appearance</value>
   </data>
+  <data name="FileExplorerPreview_ToggleSwitch_GCODE_Thumbnail.Header" xml:space="preserve">
```

Comment:
We should use the same resource name style on all strings

Reply:
Unless I'm missing something, this is using the same resource name style as for the Pdf and Svg ones!

Reply:
Okay. Didn't check that.